test/unit/GenericFactory.t.sol

```diff
@@ -11,6 +11,7 @@ import { GenericFactory } from "src/GenericFactory.sol";
 contract GenericFactoryTest is BaseTest {
     function testCreatePair_AllCurves(uint256 aCurveId) public {
         // assume
+        // TODO: Is this the way to test all curves?
         uint256 lCurveId = bound(aCurveId, 0, 1);
```

Comment:
Yes this was the way I came up with to test creating all curves

Reply:
Ahh, I forgot it was zero-indexed. Can resolve